src/test/java/BaseTest.java

```diff
@@ -92,7 +93,11 @@ public WebDriver pickBrowser(String browser) throws MalformedURLException {
                 return lambdaTest();
             default:
                 WebDriverManager.chromedriver().setup();
-                return driver = new ChromeDriver();
+                ChromeOptions co = new ChromeOptions();
+                co.addArguments("--remote-allow-origins=*");
+                co.addArguments("--window-size=1920,1080");
+                co.addArguments("--start-maximized");
+                return driver = new ChromeDriver(co);
```

Comment:
Please move this methods in LoginPage
Base Test in the main class which will take care of Browser setup, loading configuration file and other reusable methods like screenshot, handling sync issues and many more.

For example

![image](https://user-images.githubusercontent.com/104761482/235594473-7c2f8a84-ec59-441d-90f5-681c97501003.png)